src/python/aqueduct_executor/operators/connectors/data/main.py

```diff
@@ -12,4 +16,18 @@
     spec_json = base64.b64decode(args.spec)
     spec = parse_spec(spec_json)
 
+    print_with_color(
+        "Starting %s job: %s" % (spec.type.value, spec.name), color=PrintColorType.GREEN
+    )
+    begin = time.time()
+
     execute.run(spec)
+
+    end = time.time()
+    performance = {
+        "job": spec.name,
```

Comment:
We can file this as a follow-up task, but we should really print the operator name here (we'll have to pass it in through the job spec). The spec name is generally pretty useless :/